src/library/dlgtrackinfo.cpp

```diff
@@ -786,25 +786,37 @@ void DlgTrackInfo::slotImportMetadataFromFile() {
     mixxx::TrackRecord trackRecord = m_pLoadedTrack->getRecord();
     mixxx::TrackMetadata trackMetadata = trackRecord.getMetadata();
     QImage coverImage;
+
     const auto resetMissingTagMetadata = m_pUserSettings->getValue<bool>(
             mixxx::library::prefs::kResetMissingTagMetadataOnImportConfigKey);
+
+    // Pass false to skip cover loading
+    const bool importCover = false;
```

Comment:
This is a workaround for the confusing signature
`importTrackMetadataAndCoverImage()`  in case you don't want a cover. 

Is the a function that skips the cover part already? 
Probably not. Maybe it is better to invent a thin wrapper 
That only imports track metadata? This can be used in many other places in the code. Something like 
importTrackMetadata()

An alternative would be to rename the nullptr 
`const *QImage const pNoCoverImport = nullptr` 

I have not fully investigated it. Please check related code if these suggestion fits. 

If you prefer to keep the scpe limited just pass directly nullptr below.